op-guide/binary-deployment.md

```diff
@@ -0,0 +1,486 @@
+| Pushgateway |  9091 | TCP | the aggregation and report port for TiDB, TiKV, and PD monitor |
+| Node_exporter|  9100| TCP | the communication port to report the system information of every TiDB cluster node |
+| Grafana | 3000 | TCP | the port for the external Web monitoring service and client (Browser) access |
+| alertmanager | 9093 | TCP | the port for the alert service |
+
+## Configure and check the system before installation
+
+### Operating system
+
+| Configuration | Description |
+| :-- | :-------------------- |
+| Supported Platform | See the [Software and Hardware Requirements](./recommendation.md) |
+| File System  |  The ext4 file system is recommended in TiDB Deployment |
+| Swap Space  |  The Swap Space is recommended to close in TiDB Deployment  |
+| Disk Block Size  |  Set the size of the system disk `Block` to `4096` |
+
+### Network and firewall
+
+| Configuration | Description |
+| :-- | :------------------- |
+| Firewall / Port | Check whether the ports required by TiDB are accessible between the nodes |
+
+### Operating system parameters
+
+| Configuration | Description |
+| :-- | :-------------------------- |
+| Nice Limits | For system users, set the default value of `nice` in TiDB to `0` |
+| min_free_kbytes | The setting for `vm.min_free_kbytes` in `sysctl.conf` needs to be high enough |
+| User Open Files Limit | For database administrators, set the number of TiDB open files to `1000000` |
+| System Open File Limits | Set the number of system open files to `1000000` |
+| User Process Limits | For TiDB users, set the `nproc` value to `4096` in `limits.conf` |
+| Address Space Limits | For TiDB users, set the space to `unlimited` in `limits.conf` |
+| File Size Limits | For TiDB users, set the `fsize` value to `unlimited` in `limits.conf` |
+| Disk Readahead | Set the value of the `readahead` data disk to `4096` at a minimum |
+| NTP service | Configure the NTP time synchronization service for each node |
+| SELinux  | Turn off the SELinux service for each node |
+| CPU Frequency Scaling | It is recommended to turn on CPU overclocking |
+| Transparent Hugepages | For Red Hat 7+ and CentOS 7+ systems, it is required to set the Transparent Hugepages to `always` |
+| I/O Scheduler | Set the I/O Scheduler of data disks to the `deadline` mode |
+| vm.swappiness | Set `vm.swappiness = 0` |
+
+
+> **Note**: To adjust the operating system parameters, contact your system administrator.
+
+### Database running user
+
+| Configuration | Description |
+| :-- | :---------------------------- |
+| LANG environment | Set `LANG = en_US.UTF8` |
+| TZ time zone | Set the TZ time zone of all nodes to the same value |
+
+## Create the database running user account
+
+In the Linux environment, create TiDB on each installation node as a database running user, and set up the SSH mutual trust between cluster nodes. To create a running user and open SSH mutual trust, contact the system administrator. Here is an example:
```

Comment:
This is not required for TiDB communication and will be a problem for some user's security policies because it can be used to leapfrog between machines.  We should comment on what it is used for so an admin can lock it down.